e2e/test/MessageSendFaultInjectionTests.cs

```diff
@@ -18,6 +19,14 @@ public partial class MessageSendFaultInjectionTests : IDisposable
     {
         private readonly string DevicePrefix = $"E2E_{nameof(MessageSendFaultInjectionTests)}_";
         private readonly string ModulePrefix = $"E2E_{nameof(MessageSendFaultInjectionTests)}_";
+
+        private readonly int MuxWithoutPoolingDevicesCount = 2;
```

Comment:
Please create other files to handle specific scenarios such as MUX, connection pooling, etc.